mock/cpp_mock_scenarios/launch/mock_test.launch.py

```diff
@@ -75,11 +75,17 @@ def generate_launch_description():
     timeout = LaunchConfiguration("timeout", default=10)
     scenario = LaunchConfiguration("scenario", default="")
     launch_rviz = LaunchConfiguration("launch_rviz", default=False)
+    map_path = LaunchConfiguration("map_path", default=str(Path.cwd()))
+    osm_file = LaunchConfiguration("osm_file", default="lanelet2_map.osm")
     scenario_node = Node(
         package="cpp_mock_scenarios",
         executable=scenario,
         name=scenario,
         output="screen",
+        parameters=[{
```

Comment:
.osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.